packages/manager/modules/pci/src/projects/project/storages/containers/container/addUser/addUser.controller.js

```diff
@@ -0,0 +1,65 @@
+import { OBJECT_CONTAINER_USER_ROLES } from './addUser.constants';
+
+export default class PciBlockStorageDetailsAddUserController {
+  /* @ngInject */
+  constructor(
+    $translate,
+    $http,
+    CucCloudMessage,
+    PciProjectStorageContainersService,
+  ) {
+    this.$translate = $translate;
+    this.$http = $http;
+    this.CucCloudMessage = CucCloudMessage;
+    this.PciProjectStorageContainersService = PciProjectStorageContainersService;
+    this.selectedUser = null;
+    this.selectedRole = null;
+    this.addUserStep = 0;
```

Comment:
**nitpick:** Above 3 lines can be moved to `$onInit` to make the code more cleaner